doc/source/how-to-upgrade-to-flower-next.rst

```diff
@@ -328,15 +367,15 @@ Simulation in a Notebook
 
     backend_config = {"client_resources": {"num_cpus": 2, "num_gpus": 0.25}}
 
-    # Flower 1.8
+    # Flower v1.8
```

Comment:
We need to specify that from `flwr == 1.11` onwards we use the `server_fn` and `client_fn` with context. Refer to the [Use federated learning strategy](file:///Users/csng/code/flower/doc/build/html/tutorial-series-use-a-federated-learning-strategy-pytorch.html) tutorial.